app/src/main/java/com/mensinator/app/navigation/MensinatorApp.kt

```diff
@@ -145,23 +144,45 @@ fun MensinatorApp(
             navController = navController,
             startDestination = Screen.Calendar.name,
             modifier = Modifier.padding(paddingValues),
-            enterTransition = { fadeIn(animationSpec = tween(50)) },
+            enterTransition = { fadeIn(animationSpec = tween(5000)) },
             exitTransition = { fadeOut(animationSpec = tween(50)) },
         ) {//create a new file for every page and pass it inside the composable
             composable(route = Screen.Calendar.name) {
-                CalendarScreen()
+                Scaffold(
```

Comment:
Not sure whether this is correct, with this there are now two nested Scaffolds?

Reply:
yeah it works well but yeah its too a bit much. the perfect for me would be not to have a scaffold with a topbar but just a title in every screen. but we cant have the topbar change outside the nav animation

Reply:
Should we merge or look more at this?

Reply:
Nested scarfolds is possible and correct. We can accept this or we can change the scaffold topbar to just a title in every screen